packages/ui-components/src/components/single-select-dropdown/single-select-dropdown.tsx

```diff
@@ -182,7 +182,9 @@ export class KvSingleSelectDropdown implements ISingleSelectDropdown, ISingleSel
 
 	private selectOption = (selectedOption: string) => {
 		this.optionSelected.emit(selectedOption);
-		this.setOpenState(false);
+		if (!this.selectOptions.flatten[selectedOption].stub) {
```

Comment:
the stub property is optional, should we have precaution here by adding the ? operator?